translations/fr/README.md

```diff
@@ -0,0 +1,104 @@
+</div>
+<br> 
+
+Bienvenue dans le cours d'introduction à l'Open Source avec OpenSauced ! Ce cours est conçu pour vous fournir une introduction à l'open source et vous guider tout au long du processus de contribution à des projets open source.
+## Support linguistique
+Comme la pizza, l'open source est destiné à être partagé avec tout le monde. Cliquez sur les liens ci-dessous pour suivre ce cours dans les différentes langues suivantes.
+
+- [Anglais](/README.md)
+
+## Aperçu du cours
+
+Le cours est divisé en cinq chapitres, chacun couvrant un aspect différent de l'open source :
+
+1. **Qu'est-ce que l'open source?** Ce chapitre fournit une introduction à l'open source, son histoire et son importance dans le paysage actuel du développement logiciel.
+
+2. **Pourquoi open source?** Dans ce chapitre, vous découvrirez les avantages des logiciels open source, tels qu'une collaboration accrue, un développement plus rapide et une meilleure qualité.
+
+3. **Outils pour réussir.** Découvrez les outils et plates-formes clés nécessaires pour un voyage réussi dans le développement open source. De Git et GitHub pour le contrôle de version et la collaboration à CodeSee pour la visualisation de code et VS Code comme éditeur de code, ce chapitre donne un aperçu de la façon d'exploiter ces outils essentiels.
+
+4. **Comment contribuer à l'open source?**  Ce chapitre couvre la recherche de projets sur lesquels travailler, la soumission de contributions, l'intégration dans de nouveaux projets et les résultats après la soumission de contributions. Les conseils pour trouver des projets incluent la navigation sur GitHub et GitLab, le suivi de vos intérêts personnels, l'adhésion à des communautés open source et l'utilisation de plateformes comme OpenSauced.
+
+5. **La sauce secrète**  Ce chapitre se concentre sur la fourniture de stratégies et de conseils moins connus pour avoir un impact durable dans la communauté open source. Ce chapitre fournit des stratégies pour trouver des problèmes, gagner du terrain dans votre parcours open source et tirer parti d'OpenSauced pour créer votre CV open source.
+
+## Informations complémentaires
+
+Il s'agit du début de votre parcours dans le domaine de l'open source, c'est pourquoi nous avons mis à votre disposition quelques ressources supplémentaires pour vous aider à démarrer:
+- [Glossaire](09-glossarie.md): Un glossaire des termes et définitions couramment utilisés dans le domaine de l'open source.
+- [Ressources additionnelles](08-ressources-additionnelles.md): Une liste de ressources supplémentaires pour en savoir plus sur l'open source, y compris des articles de blog, des tutoriels et des guides.
+
+## Pour débuter
+
+Pour commencer le cours, il suffit de naviguer vers le fichier [`intro`](01-introduction.md) sur le dépôt et de commencer à lire ! Chaque chapitre s'appuie sur le précédent, nous recommandons donc de les lire dans l'ordre.
```

Comment:
There are 16 ` !` in here that need to be updated. The biggest problem with the spacing is depending on the screen, the punctuation might potentially fall on it's own line.